source/wp-content/themes/wporg-developer-2023/functions.php

```diff
@@ -231,6 +232,19 @@ function init() {
 	add_filter( 'wporg_canonical_url', '__return_false' );
 }
 
+/**
+ * Change the prefix of the query title on the `reference/since/x.x.x` page
+ */
+function get_reference_since_title_prefix( $prefix ) {
+	$queried_object = get_queried_object();
+
+	if ( is_archive() && $queried_object && 'wp-parser-since' === $queried_object->taxonomy ) {
```

Comment:
Can you use [is_tax( 'wp-parser-since' )](https://developer.wordpress.org/redesign-test/reference/functions/is_tax/) here to simplify this?